charts/gardener/gardenlet/charts/runtime/templates/vpa.yaml

```diff
@@ -11,4 +11,11 @@ spec:
     name: gardenlet
   updatePolicy:
     updateMode: Auto
+  {{- if .Values.global.gardenlet.resources.requests }}
+  resourcePolicy:
+    containerPolicies:
+      - containerName: '*'
+        minAllowed:
+{{ toYaml .Values.global.gardenlet.resources.requests | indent 10 }}
```

Comment:
This is only applied for manually deployed gardenlets. Did you also observe problems with the vpa for gardenlets deployed by gardener?

Reply:
> This is only applied for manually deployed gardenlets
This change would be applied to all Gardenlets as the Gardenlet in the Soil uses its own helm chart to deploy the Gardenlets the Seed.
cc @rfranzke